lib/handler/mruby.c

```diff
@@ -62,6 +62,10 @@ static void setup_globals(mrb_state *mrb)
 
     h2o_mruby_eval_expr(mrb, "$LOAD_PATH << \"#{$H2O_ROOT}/share/h2o/mruby\"");
     h2o_mruby_assert(mrb);
+
+    /* require core modules and include built-in libraries */
+    h2o_mruby_eval_expr(mrb, "require \"preloads.rb\"");
+    h2o_mruby_assert(mrb);
```

Comment:
Can we expect adequate information emitted to the log in case either requiring preloads.rb or doing the requires in preloads.rb fails?

If the answer is yes, I think we can merge this PR right away.